### PR TITLE
Fix #273: Simplified configuration for Test Server

### LINKED ISSUE #273: Simplified configuration for Test Server
Implement support for simplified configuration, see: https://github.com/wultra/powerauth-server/issues/907

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/errorhandling/DefaultExceptionHandler.java [hunk 1]
```diff
@@ -48,15 +48,27 @@ public class DefaultExceptionHandler {
...
      * @return Response with error details.
      */
     @ExceptionHandler(AppConfigNotFoundException.class)
     @ResponseStatus(HttpStatus.BAD_REQUEST)
-    public @ResponseBody ErrorResponse handleApplicationNotFoundException(AppConfigNotFoundException ex) {
-        logger.warn("Error occurred during application lookup.", ex);
-        return new ErrorResponse("APPLICATION_NOT_FOUND", "Application was not found.");
+    public @ResponseBody ErrorResponse handleAppConfigNotFoundException(AppConfigNotFoundException ex) {
+        logger.warn("Error occurred during application configuration.", ex);
+        return new ErrorResponse("APP_CONFIG_NOT_FOUND", "Application configuration was not found.");
```
Comment (banterCZ) [inline powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/errorhandling/DefaultExceptionHandler.java:59]: 🤔 We are changing the error code, but it was completely wrong anyway. We should document the values. Opening an issue for that #280

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/service/ApplicationService.java [hunk 2]
```diff
@@ -50,16 +53,39 @@ public ApplicationService(TestConfigRepository appConfigRepository) {
...
+        final String applicationSecret;
+        final String masterPublicKey;
+
+        if (mobileSdkConfig != null) {
+            final SdkConfiguration config;
+            try {
+                config = SdkConfigurationSerializer.deserialize(mobileSdkConfig);
+            } catch (Exception ex) {
+                logger.warn("Invalid mobile SDK configuration", ex);
+                throw new AppConfigInvalidException("Invalid mobile SDK configuration");
```
Comment (banterCZ) [inline powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/service/ApplicationService.java:76]: I suggest keeping the cause and logging the whole stack trace only in the handler.
```suggestion
                logger.warn("Invalid mobile SDK configuration: {}", ex.getMessage());
                throw new AppConfigInvalidException("Invalid mobile SDK configuration", ex);
```

**Review by banterCZ — APPROVED** [review]
Looks good, just a little comment about exception handling to consider.

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/errorhandling/DefaultExceptionHandler.java (hunk 1 above)
Comment (romanstrobl) [reply powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/errorhandling/DefaultExceptionHandler.java:59]: Yes, we should document more than just error codes.

**Pull request merged**